### ISSUE TITLE
Who's streaming right now?

### ISSUE THREAD

**Issue by MojoJojo86** [bnb/awesome-developer-streams#120]
I imagine the majority of code streams do so on twitch/youtube.

What do people think about taking this awesome repo even further and having a little page that realtime displays which streamers are currently streaming right now. Could categorise based on stream title if language included?

**Comment by MojoJojo86** [comment]
Yeah exactly, seems like it would be a good way to not only get to see more streams but increase viewer count on those dedicated to streaming

**Comment by lannonbr** [comment]
@MojoJojo86 I am going to start building a project for this: https://github.com/lannonbr/TwitchDevStreamExplorer

**Comment by itsdamslife** [comment]
You should add Chris Griffing https://www.twitch.tv/cmgriffing

**Comment by surlydev** [comment]
CodebaseAlpha is working on a bot so you can find the schedule of streamers. There should be some collaboration with him

**Comment by surlydev** [comment]
I added some Twitch categories to the end of the listing, to show who is streaming in Programming, Game, Mobile, Software and Web Development categories. There is also a link to the Science and Technology category.

**Comment by surlydev** [comment]
I've submitted a ticket to Twitch to ask them whether this information would be available via an API.

**Comment by andyli** [comment]
#193 adds Twitch streaming status badges.

If you're interested in how to access the info from Twitch API, see https://github.com/badges/shields/pull/3683.

**Comment by jasonwilliams** [comment]
@surlydev is there a link to this ticket?

**Comment by MarkoShiva** [comment]
The main problem of having that inside of the repo is that you can't use jekyll for that as its static pages. 
So there there should be a website hosted on github like a pages that would be built in python with js for live updating of the streams so we dont need to refresh all the time to see who is online.
